includes/class-wc-payments-order-success-page.php

```diff
@@ -73,6 +118,35 @@ public function show_woopay_payment_method_name( $payment_method_title, $abstrac
 		return ob_get_clean();
 	}
 
+	/**
+	 * Add the BNPL logo to the payment method name on the order received page.
+	 *
+	 * @param WC_Payment_Gateway_WCPay $gateway the gateway being shown.
+	 * @param WCPay\Payment_Methods\UPE_Payment_Method $payment_method the payment method being shown.
+	 *
+	 * @return string|false
+	 */
+	public function show_bnpl_payment_method_name( $gateway, $payment_method ) {
+		$method_logo_url = apply_filters(
+			'wc_payments_thank_you_page_bnpl_payment_method_logo_url',
+			$payment_method->get_payment_method_icon_for_location( 'checkout', false, $gateway->get_account_country() ),
+			$payment_method->get_id()
+		);
+
+		// If we don't have a logo URL here for some reason, bail.
+		if ( ! $method_logo_url ) {
+			return false;
+		}
+
+		ob_start();
+		?>
+		<div class="wc-payment-gateway-method-logo-wrapper wc-payment-bnpl-logo <?php echo $payment_method->get_id(); ?>">
```

Comment:
```suggestion
		<div class="wc-payment-gateway-method-logo-wrapper wc-payment-bnpl-logo">
```
On further thought, do we even need this class here or is this for some future use-case? I don't have strong feelings about removing this or leaving it here, just wondering for my own curiosity.

Reply:
I added it for future use or in the case that a particular logo needs specific CSS to make it work with whatever theme is in use.